### ISSUE TITLE
Tectonic Installer detects old installation as if it was the current one

### ISSUE THREAD

**Issue by urzds** [coreos/tectonic-forum#30]
### Tectonic Version

1.5.2-tectonic.1

### Environment

What hardware/cloud provider/hypervisor is being used with Tectonic?

bare-metal

### Expected Behavior

Tectonic Installer knows whether it installed a node or whether it does not actually belong to the cluster being currently installed.

### Actual Behavior

The installer claims that Kubernetes is up and running, even though it did not install that, yet. The node was not even booted via PXE, yet.

This can easily create confusion, since with more than a few nodes and a buggy BIOS, that has problems to boot from PXE in one go, the user will quickly loose track of which nodes are already running the current installation and which are not yet.

### Reproduction Steps

  1. Have a Kubernetes API server running on the controller from a previous installation.
  2. Run the Tectonic Installer till you reach the "Boot Cluster" stage.
  3. See that the controller is being listed with a green checkmark and "Kubernetes is running"

### Other Information

I would expect the Tectonic Installer to hand out some IDs to the nodes arranges to boot and check whether those it finds to be reachable actually carry these IDs. This way it could distinguish "some other/old installation" from "current installation".

**Comment by mfburnett** [comment]
Hey @urzds - thanks for filing this issue. Will look into this and get back to you in the next few days!

**Comment by mfburnett** [comment]
cc @ivancherepov

**Comment by dghubble** [comment]
Yes, the Tectonic Installer simply looks at the given nodes for a few services to determine whether you can proceed. There is no notion of a unique name of the particular cluster instance. So if you input nodes from an existing cluster, it will misleadingly say everything is ready.

To change the role of a machine (e.g. become part of some new cluster, perhaps not even Kubernetes), the current approach is to re-provison using out of band management to request those machines PXE boot. Then they'll install CoreOS and provision themselves anew based on the new configs.

Thanks for the suggestion.
